src/reynir/simpletree.py

```diff
@@ -2341,7 +2341,7 @@ def skipstring() -> str:
                 # !!! TODO: There should be an escape character
                 # !!! for parentheses here
                 p += 1
-            if txt[start:p].endswith("\\"):
+            if p and txt[p-1] == "\\":
```

Comment:
What happens if there is a legitimate backslash (i.e. not an escape) in the text?

Reply:
Tried it, it doesn't seem to matter.